components/test_raftstore/src/cluster.rs

```diff
@@ -1118,8 +1118,12 @@ impl<T: Simulator> Cluster<T> {
         loop {
             self.reset_leader_of_region(region_id);
             let cur_leader = self.leader_of_region(region_id);
-            if cur_leader == Some(leader.clone()) {
-                return;
+            if let Some(ref cur_leader) = cur_leader {
```

Comment:
What's wrong with the original implement?

Reply:
Previous, only `Voter` peer can become leader, so directly compare is ok, but with joint consensus, `IncommingVoter` peer and `DemotingVoter` peer can also become leader.